src/VaccineTracker/store/vaccineTracker.reducer.ts

```diff
@@ -1,19 +1,15 @@
-import {
-  GETVACCINEINFO,
-  ONAPISUCCESS,
-  ONAPIFAILURE,
-} from "./vaccineTracker.types";
+import { GETVACCINEINFO, ONAPISUCCESS } from "./vaccineTracker.types";
 
 interface vaccineReducer {
```

Comment:
When there is only one endpoint this isn't too important but for the general learning purpose this might be interesting.

I have seen quite often lately that loading, success, error states are put under one property (https://github.com/tannerlinsley/react-query/blob/master/src/core/types.ts#L264). This is often because endpoint fetching logic fulfills a state machine where one state cannot appear without the other. Here is some kind of visualisation https://xstate.js.org/viz/?gist=860f97dcf3792c75d97b9728131444e3

So I suggest to think about whether any of these three boolean will appear to be true at the same and whether they could be turn into one union type. While it looks like a very small thing using a union type instead of multiple boolean limits the amount of states and would make testing easier.